design/features/payment-links/README.md

```diff
@@ -0,0 +1,58 @@
+# Add a payment link
+
+## Status
+
+Date created: *2024-04-29* 
+
+Epic trello card: https://trello.com/c/7S8AarCY/1430-epic-add-a-payment-link  
+Mural working board: https://app.mural.co/t/gaap0347/m/gaap0347/1697117881057/d45c5ac653b9730a76915e77f257bacdcbb40047?sender=ue1ef9fc3c2ad3697c3c93132  
+___
+
+## Contents  
+
+- [Add a payment link](#add-a-payment-link)
+  - [Status](#status)
+  - [Contents](#contents)
+  - [What](#what)
+    - [User stories](#user-stories)
+  - [Why](#why)
+  - [Hypothesis](#hypothesis)
+___
+
+## What  
+
+This feature introduces new functionality to the form building tool to allow form creators to take a payment for their service. This way of taking payments is utilising the [payment page](https://payments.service.gov.uk/govuk-payment-pages/), or payment link, feature developed by GOV.UK Pay. This feature allows form fillers to pay for the service after submitting their form, making it quicker and easier to get the outcome they need.  
+
+
+### User stories  
```

Comment:
We updated these user needs to make them a bit more specific and relevant - [as per this document](https://docs.google.com/document/d/1BBwjtd2rYa3aiFieIOutynjWXWzStJup2SzA2e9Sp4c/edit#heading=h.seyd9a5yt3bm)